### DISCUSSION TITLE
SUGGESTION - Facebook message import

### DISCUSSION THREAD

**Discussion by jaybinks** [simulot/immich-go#136] [Ideas]
I wrote a very messy, POC for extracting photos / video from facebook conversations and importing them into immich.
(using your API code)

https://github.com/jaybinks/immich_facebook_messages_photos_import

**Comment by simulot** [comment]
Nice! I don't have an Facebook account to do the test. Could you provide a sample of what's is delivered by Facebook?

**Comment by jaybinks** [comment]
![Screen Shot 2024-01-09 at 10 42 20 am](https://github.com/simulot/immich-go/assets/1509627/2ef87407-6fac-4de0-bdc8-c1494d4f37b6)
Directory structure

**Comment by jaybinks** [comment]
[message_1.json](https://github.com/simulot/immich-go/files/13867599/message_1.json)
Example JSON file

**Comment by simulot** [comment]
Thanks, this is much simpler than google photo.
How to you see the import?

All photos imported in a album "Facebook import", of with the message title
What do you expect to do with the message participants?
